src/main/java/thanos/Launcher.java

```diff
@@ -0,0 +1,29 @@
+package thanos;
+
+import javafx.application.Application;
+import thanos.ui.Main;
```

Comment:
It would be better if imports from different packages are separated.
```suggestion
import javafx.application.Application;

import thanos.ui.Main;
```